src/components/app/LandingPage.js

```diff
@@ -32,22 +59,32 @@ class LandingPage extends React.Component {
             ["maintenance", i18n.t("Maintenance"), "settings"],
         ];
         const menuItems = items.map(([key, title, icon]) => (
-            <MenuItem
+            <GridListTile
                 key={key}
                 data-test={`page-${key}`}
                 onClick={this.onClick.bind(this, key)}
                 component={Link}
                 to={`/${key}`}
+                className={classes.gridTile}
             >
-                <FontIcon className="material-icons">{icon}</FontIcon>
-                {title}
-            </MenuItem>
+                <div className={classes.tileContainer}>
+                    <FontIcon
+                        className="material-icons"
+                        style={{ fontSize: 60, marginBottom: 20, color: lightGray }}
```

Comment:
As a rule of thumb, props should be static to avoid unnecessary re-renders. Unfortunately in JS, `{x: 1} === {x: 1}` is false, but we can just move those to the styles object.